src/components/api.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+const gamesApi = axios.create({
+  baseURL: "https://games-toby-w.herokuapp.com/api",
+});
+
+export const getReviews = (review_id, category) => {
```

Comment:
Are you actually using the category parameter anywhere here? Or have you added it to use later?